src/main/java/de/bonndan/nivio/input/SourceReferencesResolver.java

```diff
@@ -15,18 +17,21 @@
  */
 public class SourceReferencesResolver {
```

Comment:
Meine IDE schlägt vor, dass die Klasse package-private sein könnte.

Reply:
I'd rather keep everyting public in the input package for now. (see #14 etc)